web/content/docs/tools/getting-started/first_steps/index.pandoc

```diff
@@ -0,0 +1,26 @@
++++
+date = "2019-03-14T11:00:13+01:00"
+title = "First Steps"
+author = "Joerg Buchwald"
+
+[menu]
+  [menu.tools]
+    parent = "getting-started"
++++
+
+## Frist Steps
+
+This section describes a possible general pre-processing workflow using some of the tools provided by OGS.
+
+To set up a model domain, the project file `*.prj` requires additional input files for the mesh and geometry to apply the boundary conditions accordingly.
+
+The mesh must be provided in VTK's `*.vtu` format, whereas the geometry can be provided either in OGS' internal `*.gml` file format (not to be confused with the Geography Markup Language) or as a `*.vtu` file as well, containing boundary elements only (Multiple `*.vtu` files can be provided in the project file using the `<meshes>` tag). We recommend the usage of the first method for simple 2D meshes with constant boundary conditions, whereas more complicated geometries and conditions might require the latter method. One general advatage in the utilization of `*.vtu` files is that they allow for defining additional field variables at each mesh node/element in order to implement spatially varying boundary conditions in a similar manner as defining inhomogeneous meterial properties.
```

Comment:
spelling: advantage.
wording: "allow for defining additional" -> "allow a definition of additional"

Reply:
thx